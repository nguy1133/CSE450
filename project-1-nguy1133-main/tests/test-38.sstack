// ERROR: CONCAT cannot have more than two arguments.
CONCAT "Too many" " args " "to concatenate"
