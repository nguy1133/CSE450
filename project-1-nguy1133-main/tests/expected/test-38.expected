ERROR (line 2): Unexpected token '"to concatenate"'
